include/crow/websocket.h

```diff
@@ -626,17 +656,21 @@ namespace crow
                     }
                     asio::async_write(
                       adaptor_.socket(), buffers,
-                      [&](const asio::error_code& ec, std::size_t /*bytes_transferred*/) {
-                          sending_buffers_.clear();
+                      [&, watch = std::weak_ptr<void>{anchor_}](const asio::error_code& ec, std::size_t /*bytes_transferred*/) {
                           if (!ec && !close_connection_)
```

Comment:
as well

Reply:
Okay, however, as it confilicts due to some changes on `websocket::post` and `dispatch`, I'll make changes for this after any decision is made for #535.